lib/cogctl/actions/groups/create.ex

```diff
@@ -1,36 +1,28 @@
 defmodule Cogctl.Actions.Groups.Create do
   use Cogctl.Action, "groups create"
-  alias Cogctl.Table
+
+  alias Cogctl.Actions.Groups
```

Comment:
if you do `import Cogctl.Actions.Groups, only: [render: 2]`, you can just call `render(group, message)` below